internal/commands/push.go

```diff
@@ -127,7 +129,11 @@ func runPush(dockerCli command.Cli, name string, opts pushOptions) error {
 		return errors.Wrapf(err, "fixing up %q for push", retag.cnabRef)
 	}
 	// push bundle manifest
-	descriptor, err := remotes.Push(context.Background(), bndl, retag.cnabRef, resolverConfig.Resolver, true, withAppAnnotations)
+	logger := logrus.New()
+	logger.SetLevel(logrus.FatalLevel)
```

Comment:
I wonder if we don't want error level too? 🤔

Reply:
Done